saleor/checkout/base_calculations.py

```diff
@@ -192,7 +192,12 @@ def base_checkout_total(
     discounts: Iterable[DiscountInfo],
     lines: Iterable["CheckoutLineInfo"],
 ) -> Money:
-    """Return the total cost of the checkout."""
+    """Return the total cost of the checkout.
+
+    The price includes sales, specific voucher and applied once per order
+    voucher discounts.
+    The price does not include the entire order discount.
```

Comment:
Don't know if this is already done, but maybe it would be worth having a test, that will fail if someone change the `base_checkout_total` to include voucher.entire order?

Reply:
What about voucher for shipping? Maybe would be worth to mention it here also

Reply:
I don't understand what kind of test you want to add 🤔.

Reply:
I will update the description

Reply:
Ok, I get what you mean. But there is already a test that will fail because of that